hive/src/main/java/com/netease/arctic/hive/catalog/ArcticHiveCatalog.java

```diff
@@ -165,6 +165,7 @@ public TableBuilder withPrimaryKeySpec(PrimaryKeySpec primaryKeySpec) {
     public TableBuilder withProperty(String key, String value) {
       if (key.equals(HiveTableProperties.ALLOW_HIVE_TABLE_EXISTED) && value.equals("true")) {
         allowExistedHiveTable = true;
+        super.withProperty(key, value);
```

Comment:
Why do we need to pass this parameter up?

Reply:
Because in `MixedHiveTables.allowExistedHiveTable` is through the table properties allow - hive - table - existed attribute to determine whether to allow hive table, So here we need to write allow-hive-table-existed into table properties.